src/context.rs

```diff
@@ -199,11 +202,26 @@ impl Game {
                         *count -= 1;
                     }
                 }
+                ScheduledStop::Instruction(opcode) => {
+                    if let Some(current_opcode) = &self.cpu.controller.opcode {
+                        let was_prefixed =
+                            matches!(current_opcode, OpcodeType::CBPrefixed(_opcode));
+                        let current_opcode = current_opcode.to_string();
+                        let opcode_changed = *opcode != current_opcode;
+                        if opcode_changed && !was_prefixed {
```

Comment:
Why the expection on `CB` prefixed opcode ?

Reply:
Because to get the next opcode with this implementation, we check if the opcode changed in the controller. Therefore, without an exception for `CB` prefixed opcodes, we have to click 2 times to reach the next instruction.

Reply:
Ok